### ISSUE TITLE
What about Immersive Space ?

### ISSUE THREAD

**Issue by dyunchik** [steventroughtonsmith/VisionVolumetricUIKit#1]
How to add Immersive Space in UIKit lifecycle app?

**Comment by steventroughtonsmith** [comment]
I don't believe there's a way to do any of the immersive space stuff from a UIKit-based lifecycle, but it's not something I've looked into
